pages/linux/arpaname.md

```diff
@@ -0,0 +1,8 @@
+# arpaname
+
+> Provides corresponding ARPA name for a IP address.
+> More information: <https://manned.org/arpaname>.
+
+- Translate IP addresses (IPv4 and IPv6) to the corresponding ARPA name:
+
+`arpaname {{1.2.3.4}}`
```

Comment:
@navarroaxel Can this placeholder be an example or should we mention ip address?

Reply:
`ip_address` is a better placeholder, otherwise this only represents a IPv4 example
```suggestion
`arpaname {{ip_address}}`
```